lib/customisation.nix

```diff
@@ -365,6 +378,35 @@ rec {
     else
       mapAttrs mkAttrOverridable pkgs;
 
+  makeAlias =
+    let
+      currentRelease = lib.versions.majorMinor lib.version;
+    in
+    self: name:
+    {
+      type,
+      package ? null,
+      reason ? null,
+      release ? null,
+    }:
+    assert type == "alias";
```

Comment:
Seems like busywork to have to set this? It should be relatively low‐ceremony to stub out a package. `lib.optionalAttrs (!config.allowAliases)` is already annoying.